### PR TITLE
Don't add ReLU automatically

### PR DESCRIPTION
**Pull request by kristofbc**
TensorFlow-Slim use ReLU activations by default for convolution, deconvolution and fully-connected transformations.

After speaking with @cbfinn, the ReLU were unintentional. They were introduced after open-sourcing the code and using TensorFlow-Slim.

### CODE REVIEW

**Comment by kristofbc** [comment]
I signed it!

File: video_prediction/prediction_model.py [hunk 1]
```diff
@@ -172,25 +173,26 @@ def construct_model(images,
...
+            enc6, color_channels, 1, stride=1, scope='convt4', activation_fn=None)
         # This allows the network to also generate one image from scratch,
         # which is useful when regions of the image become unoccluded.
         transformed = [tf.nn.sigmoid(enc7)]
 
       if stp:
         stp_input0 = tf.reshape(hidden5, [int(batch_size), -1])
         stp_input1 = slim.layers.fully_connected(
-            stp_input0, 100, scope='fc_stp')
+            stp_input0, 100, scope='fc_stp', activation_fn=None)
```
Comment (cbfinn) [inline video_prediction/prediction_model.py]: should still have a ReLU here.

Comment (kristofbc) [inline video_prediction/prediction_model.py]: Thank you! I've change this line back to the original where the ReLU activation was default.

**Comment by cbfinn** [comment]
LGTM

**Pull request closed without merging**